scripts/core/virtual memory.yml

```diff
@@ -0,0 +1,282 @@
+#
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
```

Comment:
Just wanted to give an update that the license of the project have been updated in this https://github.com/oneapi-src/unified-runtime/pull/557. So, we should be able to update all the files license to the llvm one now.

Reply:
Fixed.